assets/component-slider.css

```diff
@@ -359,6 +363,39 @@ slider-component.slider-component-full-width {
   justify-content: center;
 }
 
+.slider-button--next {
+  margin-right: -2rem;
+  height: 100%;
+}
+
+.slider-button--prev {
+  margin-left: -2rem;
+  height: 100%;
+}
+
+
+@media screen and (min-width: 750px) {
+  .slider-button--next {
+    margin-right: -3.8rem;
+  }
+
+  .slider-button--prev {
+    margin-left: -3.8rem;
+  }
+
+}
+
+@media screen and (min-width: 990px) {
+  body:has( .section-header .header:not(.drawer-menu)) .announcement-bar-section .slider-button--next {
+    margin-right: -2rem;
+  }
+
+  body:has( .section-header .header:not(.drawer-menu)) .announcement-bar-section .slider-button--prev {
+    margin-left: -2rem;
+  }
+
```

Comment:
Update from #help-accessibility channel:

"If the merchant sets it to auto-start and auto-advance, prefers-reduced-motion should override both of those IMO. So the user would click through the slides."

So, it's still an opinion, however, I don't have strong feelings against/for it. I'm okay to prevent the slider from rotation in case of a user having reduce-motion-on on their computer.

Reply:
> As far as I understand , `prefers-reduced-motion` should reduce motion but not remove it.

Not sure where you read this, but that's not quite right. `prefers-reduced-motion` is a media query that does whatever you tell it to do, allowing us to include animations/motion on the page while also making sure that they don't cause any issues or discomfort to people who suffer from vestibular disorders (or to those who simply don't like a lot of motion). We actually have a good example of using this query in recent [PRs](https://github.com/Shopify/dawn/pull/2342), where we're _only turning animations on_ if the user doesn't have `prefers-reduced-motion` enabled ([see code](https://github.com/Shopify/dawn/blob/e80839333aca4357e96eb81b2a1342a950159f8a/assets/base.css#L3139-L3148)).

What that Slack thread is saying is that slides should _not_ automatically change. There is [official guidance in WCAG](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html) for this, so it's not just an opinion. It's a level A requirement. The reason why we don't have it in the slideshow is that we didn't have much motion in Dawn before. Now that we're introducing more sections with different types of animations/motion, we want to make sure it's not all too overpowering. I think that's the reason why it was added as a requirement but @stufreen, who created the issue, should be able to clarify.

Reply:
Sounds good 🎉

Reply:
It's [done](https://github.com/Shopify/dawn/pull/2394/commits/c0a1d44c1de46c226bbddb0fc4a760f8e979a187).

I also applied it to the slideshow and hide play/pause buttons when reduced-motion is on.